Changelog.md

```diff
@@ -4,9 +4,9 @@ Language Features:
 * Yul: Introduce builtin `clz(x)` for counting the number of leading zero bits in a 256-bit word.
 
 Compiler Features:
```

Comment:
I wanted to comment that we should fix the spacing between sections to empty lines, but let's just keep it like this. I constantly see people changing it back and forth in PRs for no reason. Maybe this will finally end this madness.